warehouse/configuration_testing/validate.go

```diff
@@ -147,7 +149,30 @@ func (ct *CTHandleT) verifyingCreateTable() (err error) {
 	defer ct.cleanup()
 
 	// Create table
-	return ct.createTable()
+	err = ct.createTable()
+	if err != nil {
+		return
+	}
+
+	// Alter table
+	err = ct.alterTable()
+	return
+}
+
+func (ct *CTHandleT) verifyingFetchSchema() (err error) {
+	// Getting warehouse client
+	ct.client, err = ct.whManager.Connect(ct.warehouse)
+	if err != nil {
+		return
+	}
+	// Creating create table query and running over the warehouse
+	if ct.GetDestinationType() == warehouseutils.DELTALAKE {
+		dbHandle := ct.GetDatabricksHandle()
+		_, err = dbHandle.FetchSchema(ct.warehouse)
+	} else {
+		_, err = ct.client.Query(ct.FetchSchemaQuery(), client.Read)
+	}
```

Comment:
Similar comment as the BQ one. Ideally the warehouse config testing shouldn't know nothing about the internals of destinations and the functions they use to perform the changes.

Reply:
Done.